package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this file needed?

Reply:
It's auto generated when install dependencies.